packages/notifications/src/styled/content/StyledParagraph.tsx

```diff
@@ -6,19 +6,17 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import CalloutStyles from '@zendeskgarden/css-callouts';
 import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'notifications.paragraph';
 
 /**
- * Used for multi-line Notification content. Supports all `<div>` props
+ * Used for multi-line Notification content. Supports all `<p>` props
```

Comment:
Please update the `Paragraph` element JSDoc accordingly (div -> p)

Reply:
Updated, thanks 👍!